src/dex/aave-v2/aave-v2.ts

```diff
@@ -51,6 +51,8 @@ export class AaveV2
   readonly hasConstantPriceLargeAmounts = true;
   readonly isFeeOnTransferSupported = false;
 
+  needWrapNative = false;
```

Comment:
fyi you don't need to explicitly specify `false`, as it's `false` by default - https://github.com/paraswap/paraswap-dex-lib/blob/master/src/dex/simple-exchange.ts#L41

Reply:
refactored needWrapNative a little bit, now it's possible to use default value false